### PR TITLE
update for removal of configmap varibles from enforce stack script

### PR DESCRIPTION
**Pull request by aadeshpa**
it also fix the verification for the build step in all pipelines

### CODE REVIEW

**Comment by aadeshpa** [comment]
This PR is being tested. Please do not merge yet.

**Comment by aadeshpa** [comment]
This PR is ready to be merged for 0.9.0-rc2

File: pipelines/docker/kabanero-utils/scripts/enforce_stack_policy.sh [hunk 1]
```diff
@@ -242,12 +231,7 @@
         fi
 
         # Get the target sha256 digest from the image registry. Use the proper credentials depending on what was passed to us
-        echo "$INFO RECONCILED_STACK_IMAGE_REGISTRY_HOST = $IMAGE_REGISTRY_HOST"
-        if [ -z "$IMAGE_REGISTRY_PASSWORD" ] || [ -z "$IMAGE_REGISTRY_USERNAME" ]; then
-           TARGET_DIGEST=$( skopeo inspect docker://"$IMAGE_REGISTRY_HOST"/"$PROJECT"/"$STACK_NAME":"$VERSION" | jq '.Digest' )
-        else
-           TARGET_DIGEST=$( skopeo inspect --creds="$IMAGE_REGISTRY_USERNAME":"$IMAGE_REGISTRY_PASSWORD" docker://"$IMAGE_REGISTRY_HOST"/"$PROJECT"/"$STACK_NAME":"$VERSION" | jq '.Digest' )
-        fi
+        TARGET_DIGEST=$( skopeo inspect docker://"$IMAGE_REGISTRY_HOST"/"$PROJECT"/"$STACK_NAME":"$VERSION" | jq '.Digest' )
```
Comment (kvijai82) [inline pipelines/docker/kabanero-utils/scripts/enforce_stack_policy.sh:234]: How would this now work for private registries? Do the secrets somehow get inherited by the scripts?

Comment (aadeshpa) [reply pipelines/docker/kabanero-utils/scripts/enforce_stack_policy.sh:234]: yes , when i tested with artifactory url  in past, I just created a secret via tekton dashboard and push and pull worked with internal kubernetes magic or tekton dashboard magic.

**Comment by aadeshpa** [comment]
> Would we need to do the internal registry mapping here also?

@kvijai82 : yes, this mapping is now a part of separate script named `/scripts/stack_registry_url_setup.sh` I think i should be calling this in the enforce stack scripts.

**Pull request merged**